### PR TITLE
isAllColumnOptionEnabled added

### PR DESCRIPTION
**Pull request by olhabn**
@kgrgreer @jlhughes 

Please review

small suggestion to add property that identifies either ExportDriverRegistry provides allColumn functionality or not
https://nanopay.atlassian.net/browse/NP-1401

### CODE REVIEW

File: src/foam/nanos/export/ExportDriverRegistry.js [hunk 1]
```diff
@@ -42,6 +42,10 @@ foam.CLASS({
       class: 'Boolean',
       name: 'isOpenable'
     },
+    {
+      class: 'Boolean',
+      name: 'isAllColumnOptionEnabled',
```
Comment (jlhughes) [inline src/foam/nanos/export/ExportDriverRegistry.js]: Can you add documentation to this. please.

Comment (olhabn) [reply src/foam/nanos/export/ExportDriverRegistry.js]: fixed

File: src/foam/nanos/export/ExportDriverRegistry.js [hunk 2]
```diff
@@ -42,6 +42,11 @@ foam.CLASS({
...
```
Comment (jlhughes) [inline src/foam/nanos/export/ExportDriverRegistry.js]: can this be named:
'exportAllColumns'

Comment (olhabn) [reply src/foam/nanos/export/ExportDriverRegistry.js]: fixed

**Review by jlhughes — APPROVED** [review]

**Pull request merged**